backend/src/main/java/com/capstone/moneytree/service/impl/DefaultUserService.java

```diff
@@ -57,24 +58,23 @@
     private static final String USER_CANT_BE_FOLLOWED_BY_ITSELF = "User cannot followed by itself";
 
     @Value("${alpaca.key.id}")
-    private String clientId;
```

Comment:
I think there used to be a code smell regarding this and private was added to fix it, but seems like it doesn't show up anymore so that's good.

Reply:
Yeah I've seen some SonarCloud alerts coming back from time to time. If it comes back in future PRs I'll fix it.